app/src/components/job/hr/CreateJob.jsx

```diff
@@ -36,6 +42,7 @@ export default function HRCreateJob() {
     setJobName("");
     setJobDesc("");
     setErrors([]);
```

Comment:
If seterrors is here and it is the final call, not sure if the errors will forever be empty.

Reply:
![image](https://user-images.githubusercontent.com/89028059/200043499-293c28df-5256-4cc8-81a1-0ed3ae3a674a.png)

I don't think so. I tried a few times and seemed to work well. This particular `setError()` only passes to clear previous errors when the user creates a new job, after already created one.